styles/whatsapp-web/catppuccin.user.css

```diff
@@ -94,6 +94,20 @@
     /* general background */
     --app-background: @base !important;
 
+     /*FILTER GROUP*
+    /*filter background*/
+    --filters-container-background: var(--base);
+    /* active filter text */
+    --filters-item-active-color: var(--mauve);      
+    /* active filter background */
+    --filters-item-active-background: var(--mantle);       
+    /* inactive filter background */
+    --filters-item-background: var(--surface0);
+    /* inactive filter text */
+    --filters-item-color: var(--text);
+    /* inactive filter background on hover*/
+    --filters-item-background-hover: var(--surface1);
```

Comment:
please use `@text` instead of `var(--text)` and so on for the other added variables here.